packages/ETLoad.package/ETLSegmentLoader.class/instance/loadUsing..st

```diff
@@ -14,4 +14,6 @@ loadUsing: outPointers
 	self loadFromPlaceholders: placeholderMap values usingMap: oopMap.
 	
 	"4 = oop to the first object in an image segment (bytes 0-3 = header)"
-	^ placeholderMap at: 4 ifAbsent: [ nil. ].
\ No newline at end of file
+	{ 8 . 12 . 16 } do: [ :oop |
+		placeholderMap at: oop ifPresent: [ :obj | ^ obj. ]. ].
+	^ nil.
```

Comment:
Shouldn't we rather report an error here? I believe that even an "empty" image segment will always at least contain an (empty) array of roots.